src/components/Toolbar/index.js

```diff
@@ -189,7 +190,16 @@ export default class Toolbar extends Component {
   }
 
   reveal(scopeString) {
-    this.props.onReveal(scopeString);
+    swal({
+      title: 'Are you sure you want to reveal the answer?',
+      icon: 'warning',
+      buttons: true,
+      dangerMode: true,
+    }).then((willDelete) => {
```

Comment:
hmm, maybe this should be named something else, like `shouldReveal` (or something simple like `yes`)